docs/source/api/v4/cdn_locks.rst

```diff
@@ -82,6 +82,7 @@ Allows user to acquire a lock on a CDN.
 Request Structure
 -----------------
 The request body must be a single ``CDN Lock`` object with the following keys:
+
```

Comment:
This fix also needs to be applied to the APIv5 docs for this endpoint in order to fix #7221

Reply:
done